db/migrate/20230919171846_create_likes.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :author, foreign_key: { to_table: :users }
```

Comment:
- [ ] Same here, The foreign key between the likes table and the users table should be `user_id`, not `author_id` as you did. Please fix that. Check the given ERD below.

Reply:
Thank you so much, @Omar-Muhamad , for the review. I'll work on the changes right away!